new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleDomainValidation.java

```diff
@@ -41,6 +49,49 @@ public void testCreatingDomain() {
         "Failed to create unique namespace due to ApiException");
     logger.info("Got a new namespace called {0}", namespace);
 
+    // create persistent volume and persistent volume claim
+    final String pvcName = domainUID + "-pvc"; // name of the persistent volume claim
+    final String pvName = domainUID + "-pv"; // name of the persistent volume
+
+    V1PersistentVolumeClaim v1pvc = new V1PersistentVolumeClaim()
+        .spec(new V1PersistentVolumeClaimSpec()
+            .addAccessModesItem("ReadWriteMany")
+            .storageClassName(domainUID + "-weblogic-domain-storage-class")
+            .volumeName(domainUID + "-weblogic-pv")
+            .resources(new V1ResourceRequirements()
+                .putRequestsItem("storage", Quantity.fromString("10Gi"))))
+        .metadata(new V1ObjectMetaBuilder()
+            .withName(pvcName)
+            .withNamespace(namespace)
+            .build()
+            .putLabelsItem("weblogic.resourceVersion", "domain-v2")
+            .putLabelsItem("weblogic.domainUID", domainUID));
+
+    boolean success = assertDoesNotThrow(
+        () -> TestActions.createPersistentVolumeClaim(v1pvc)
+    );
+    assertTrue(success, "PersistentVolumeClaim creation failed");
+
+    V1PersistentVolume v1pv = new V1PersistentVolume()
+        .spec(new V1PersistentVolumeSpec()
+            .addAccessModesItem("ReadWriteMany")
+            .storageClassName(domainUID + "-weblogic-domain-storage-class")
+            .volumeMode("Filesystem")
+            .putCapacityItem("storage", Quantity.fromString("10Gi"))
+            .persistentVolumeReclaimPolicy("Recycle")
+            .hostPath(new V1HostPathVolumeSource()
+                .path(System.getProperty("java.io.tmpdir") + domainUID + "-persistentVolume")))
+        .metadata(new V1ObjectMetaBuilder()
+            .withName(pvName)
+            .withNamespace(namespace)
+            .build()
+            .putLabelsItem("weblogic.resourceVersion", "domain-v2")
+            .putLabelsItem("weblogic.domainUID", domainUID));
+    success = assertDoesNotThrow(
+        () -> TestActions.createPersistentVolume(v1pv)
```

Comment:
Add message string for when it does throw an exception.  What should the engineer debugging this look for as the problem?

Reply:
Added message